### PR TITLE
Material Design Lite

### PR DESCRIPTION
**Pull request by amasciulli**
Introduce some Material Design thanks to http://www.getmdl.io/

screenshot

### CODE REVIEW

**Comment by fbarthelery** [comment]
LGTM :+1:

**Comment by calvarez-geny** [comment]
Pretty :)
Can the title be left-aligned?
Also, what about "last activity"?  Maybe you need to rebase to have that column.

File: gm_pr/templates/pr.html [hunk 1]
```diff
@@ -17,64 +17,88 @@
...
+                <td>
+                    {% for label in pr.labels %}
+                        <span class="label" style="background-color:#{{label.color}}">{{label.name}}</span>
+                    {% endfor %}
                 </td>
                 <td class="title"><a href="{{pr.url}}">{{pr.title}}</a></td>
                 <td class="user">{{pr.user}}</td>
                 <td class="nbrev">{{pr.nbreview}}</td>
-                {% if request.GET.login %}
-                <td class="my-open-comment-count">
```
Comment (calvarez-geny) [inline gm_pr/templates/pr.html]: why did you remove the "comment count" column?

**Comment by calvarez-geny** [comment]
The labels don't appear?
For the alignment, could we have?
last update: right
reviews & feedback columns: center
other columns: left
?

**Comment by amasciulli** [comment]
@caarmen I made this screenshot before merging your last changes with Last Activity, so your column is here. I'll change the alignments in a future PR.

File: gm_pr/templates/pr.html (hunk 1 above)
Comment (amasciulli) [inline gm_pr/templates/pr.html]: oops

**Comment by calvarez-geny** [comment]
Can you put a new screenshot please?  I'm also concerned about no labels appearing.

**Comment by amasciulli** [comment]
I'll make a new screenshot, but labels do appear. It's just that there are no labels on any PR at https://github.com/android/platform_frameworks_base/pulls

**Comment by calvarez-geny** [comment]
Can you configure your local project to look at this set of prs instead?
https://github.com/google/material-design-lite/pulls

It has labels and milestones

**Comment by calvarez-geny** [comment]
And to test the "my open comments" column, you can append this to the url (of your gm_pr instance):
> &login=surma

**Comment by calvarez-geny** [comment]
Thanks for the new screenshot :)

If it's too complicated to do separate alignment per column (I think it shouldn't be too hard, as there are already separate css styles for the feedback columns), could you at least make the default alignment left-aligned?

File: gm_pr/templates/pr.html [hunk 2]
```diff
@@ -17,64 +17,88 @@
...
+
+
 {% block content %}
 
-{% for project in project_list %}
-<div class="project-box">
-    <div class="project">
-        <div class="project-name">{{project.name}}</div>
-        <table class="project">
+<style>
```
Comment (calvarez-geny) [inline gm_pr/templates/pr.html]: Can we keep a gm_pr.css file for the custom css styles?

File: web/static/gm_pr.css [hunk 3]
```diff
@@ -0,0 +1,28 @@
+footer {
```
Comment (calvarez-geny) [inline web/static/gm_pr.css]: Thanks for isolating the css.

The file needs copyright.

File: gm_pr/templates/pr.html [hunk 4]
```diff
@@ -17,64 +17,76 @@
...
                 </td>
-                <td class="milestone">{% if pr.milestone %}{{pr.milestone}}{% endif %}</td>
-                <td class="label">
-                {% for label in pr.labels %}
-                    <span class="labelname labelname-{{label.style}}" style="background-color:#{{label.color}}">{{label.name}}</span>
-                {% endfor %}
+                <td class="alignleft">{% if pr.milestone %}{{pr.milestone}}{% endif %}</td>
+                <td class="alignleft">
+                    {% for label in pr.labels %}
+                        <span class="label" style="background-color:#{{label.color}}">{{label.name}}</span><br>
```
Comment (calvarez-geny) [inline gm_pr/templates/pr.html]: Need to keep the ```labelname-{{label.style}}``` to make sure dark labels have a light text

Comment (calvarez-geny) [inline gm_pr/templates/pr.html]: With this part from the old gm_pr.css:
```
-.labelname {
-    font-size: 0.7em;
-    box-shadow: 1px 1px 2px #333;
-    padding: 1px;
-}
-.labelname-dark {
-    color: #fff;
-}
-.labelname-light {
-    color: #000;
-}
```

**Comment by calvarez-geny** [comment]
Nice!  Thanks for this cool makeover.
:+1:
